src/components/ContributionButtonWithReminder.tsx

```diff
@@ -76,14 +83,20 @@ const PrimaryButton = ({ buttonUrl, buttonText, trackClick }: PrimaryButtonProps
 );
 
 interface RemindMeButtonProps {
+    disabled: boolean;
```

Comment:
I'm a bit confused about when this button will be disabled? I can see that the following is passed:

```
reminderStatus !== ReminderStatus.Editing
```

What does `ReminderStatus.Editing` mean here? (i.e. what state does it represent?) Would it be fair to think of `Editing` as the default state when a user hasn't taken any action (yet)?

Reply:
See comment below - I don't think we need this functionality, which is used in RRCP Banner/Epic for signed-out users who have to manually enter their email address.

Reply:
We no longer use this journey flow, thus suggested fix no longer required